admin/tests/unit/components/certification-details-competence-test.js

```diff
@@ -26,11 +26,6 @@ module('Unit | Component | certification-details-competence', function(hooks) {
     };
   };
 
-  test('it exists', function(assert) {
-    const component = this.owner.factoryFor('component:certification-details-competence').create();
-    assert.ok(component);
-  });
```

Comment:
Par contre quand y a d'autres tests dans le fichier je suis pour enlever les 'it exists'